reference/sql_workarounds.rst

```diff
@@ -0,0 +1,141 @@
+.. _workarounds:
+
+SQL Workarounds
+===============
+
+Citus supports most, but not all, SQL statements directly. Its SQL support continues to improve. Also many of the unsupported features have workarounds; below are a number of the most useful.
+
+Subqueries
+----------
+
+In analytics it is common to model information using a "star schema." This is a collection of dimensional tables (e.g. observations, statistics) referencing a table of facts (e.g. people, pages or events). Furthermore, a common form of query examines dimensions for a subset of the facts. Examples of these kind of queries:
```

Comment:
@begriffs  I sent the email. We came up with some common 3 patterns(2 table - preventing repartition joins, single users table and segmenting the users based on 2 events which cannot be mapped to 1 row, single table and doing aggregates/lookups over first n[based on another aggregate] rows  )when we see Subqueries in Where clause. Could you edit the documentation based on these ideas?